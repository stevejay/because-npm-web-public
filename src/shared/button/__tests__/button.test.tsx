import Button, { ButtonType } from "../button";
import React from "react";
import { cleanup, render } from "react-testing-library";
import { noop } from "lodash";

afterEach(cleanup);

test("displays correctly", async () => {
  const { container } = render(
    <Button type={ButtonType.Primary} onClick={noop}>
      The text
    </Button>
  );
  expect(container).toMatchSnapshot();
});
