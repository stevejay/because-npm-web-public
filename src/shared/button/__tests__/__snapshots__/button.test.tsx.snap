// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays correctly 1`] = `
<div>
  <button
    class="button primaryButton"
  >
    The text
  </button>
</div>
`;
